pkg/operations/upgradecluster.go

```diff
@@ -1,52 +1,73 @@
 package operations
 
 import (
-	"github.com/Azure/acs-engine/pkg/api/vlabs"
-	"github.com/satori/uuid"
+	"fmt"
+
+	"strings"
+
+	"github.com/Azure/acs-engine/pkg/api"
+	"github.com/Azure/acs-engine/pkg/operations/armhelpers"
+	"github.com/Azure/azure-sdk-for-go/arm/compute"
+	"github.com/prometheus/common/log"
```

Comment:
Yes, long terms we need to pull stuff from the RP but this is a good start, makes it functional at least.